src/app/Providers/components/VMwareProviderHostsTable/SelectNetworkModal.tsx

```diff
@@ -51,32 +63,38 @@ const SelectNetworkModal: React.FunctionComponent<ISelectNetworkModalProps> = ({
       ]}
     >
       <Form>
-        <FormGroup label="Host admin userid" isRequired fieldId="admin-userid">
-          <TextInput id="admin-userid" value={adminUserId} type="text" onChange={setAdminUserId} />
-        </FormGroup>
-        <FormGroup label="Host admin password" isRequired fieldId="admin-password">
-          <TextInput
-            id="admin-password"
-            value={adminPassword}
-            type="password"
-            onChange={setAdminPassword}
-          />
-        </FormGroup>
+        <ValidatedTextInput
+          field={form.fields.adminUserId}
+          label="Host admin userid"
+          isRequired
+          fieldId="admin-userid"
+        />
+        <ValidatedTextInput
+          field={form.fields.adminPassword}
+          label="Host admin password"
+          isRequired
+          fieldId="admin-password"
+        />
         <FormGroup
           label="Network"
           isRequired
           fieldId="network"
-          helperTextInvalid="TODO"
-          validated="default" // TODO add state/validation/errors to this and other FormGroups
+          helperTextInvalid={form.fields.selectedNetwork.error}
+          validated={form.fields.selectedNetwork.isValid ? 'default' : 'error'}
```

Comment:
If you have `getFormGroupProps` spread here you don't need the `helperTextInvalid` and `validated` props, they are provided by `getFormGroupProps`.